react/README.md

```diff
@@ -28,46 +29,71 @@ This course is aimed at the students of the RS School who have passed RS School
 
 #### Practical tasks
 
-[Task 1](modules//tasks/class-components.md)
+[Task 1](modules/tasks/class-components.md)
+
+- Cross-check review: [Task 1](modules/tasks/class-components.md)
+- Mentor review: [Task 1](modules/tasks/class-components.md)
```

Comment:
```suggestion
```
There is nothing to review. The students only work on task 1. And we discussed that the first task only cross-check because not all students should have the mentor yet.